### DISCUSSION TITLE
I've got my Developer Student Pack approved but I can't access it.

### DISCUSSION THREAD

**Discussion by MrDaPoyo** [community/community#77707] [GitHub Education]
### Hi everyone,

I've already been waiting for more than 3 days, the expected time for activating. What should I do?

**Comment by mesh-dell** [comment]
Please try the following steps to troubleshoot the issue:

1. Check your eligibility: Make sure you meet the eligibility criteria for the Developer Student Pack. You must be a student enrolled in a degree program at an accredited educational institution and have a valid student email address.

2. Verify your account: Ensure your GitHub account is verified by confirming your email address. You can do this by clicking on the verification link sent to your email address or by going to your GitHub account settings and clicking the "Verify email" button.

3. Check your email: Check your email inbox for any messages from GitHub regarding your Developer Student Pack benefits. Sometimes, the activation email may be sent to a spam or promotions folder.

4. Clear your browser cache and cookies: Sometimes, cached data or cookies can interfere with accessing your benefits. Try clearing your browser's cache and cookies and then reloading the GitHub page.

5. Check for known issues: Check the GitHub Developer Student Pack FAQ page or the GitHub status page for any known issues that may be affecting access to benefits.

6. Contact GitHub support: If none of the above steps resolve the issue, contact GitHub support for further assistance. They can help you investigate your specific case and provide personalized troubleshooting steps
